ResearchKit/+Medable/ORKFormItemCell+Medable.m

```diff
@@ -0,0 +1,95 @@
+//
+//  ORKFormItemCell+Medable.m
+//  Axon
+//
+//  Created by me on 5/18/18.
+//  Copyright © 2018 Medable Inc. All rights reserved.
+//
+
+#import "ResearchKit.h"
+#import "ORKFormItemCell.h"
+
+@interface ORKFormItemCell ()
+- (void)ork_setAnswer:(id)answer;
+@end
+
+#pragma mark -
+
+@implementation ORKFormItemTextFieldBasedCell (Medable)
+
+- (void)ork_setAnswer:(id)answer
+{
+    [super ork_setAnswer:answer];
+    
+    @try
```

Comment:
only the code generating an exception should be inside the try block

Reply:
the try/catch is to safely handle the KVC/performSelector calls failing, which are used to avoid adding a hard dependency to the `DBZxcvbn` library and the properties added to `MDRTextAnswerFormat`. The results of those calls are used w/in this try block.